### PR TITLE
propose fix to #82

### PR DESCRIPTION
**Pull request by drmowinckels**

### CODE REVIEW

File: R/meetup_query.R [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#' General Meetup query
+#'
+#' Create your own query to the Meetup API
+#' (https://www.meetup.com/meetup_api/docs/).
+#'
+#'
+#' @param api_path url path to send the meetup query after the 'https://api.meetup.com/'
+#' @param ... additional parameters to the query
+#'
+#' @return raw list results from the query, untitied
```
Comment (maelle) [inline R/meetup_query.R]:
```suggestion
#' @return raw list results from the query, untidied
```

Is untidied actually correct? :thinking:

File: R/meetup_query.R [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#'
+#' meetup_query("rladies-oslo")
+#'
+#' meetup_query("rladies-oslo", event_statue = "past")
```
Comment (maelle) [inline R/meetup_query.R]:
```suggestion
#' meetup_query("rladies-oslo", event_status = "past")
```

File: R/meetup_query.R [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
+#' General Meetup query
+#'
+#' Create your own query to the Meetup API
```
Comment (maelle) [inline R/meetup_query.R:3]: Add a comment about this being useful for endpoints we don't support with high-level helpers yet?

**Review by maelle — CHANGES_REQUESTED** [review]
Thanks a ton! Could you please run `devtools::document()`?

File: R/meetup_query.R (hunk 3 above)
Comment (drmowinckels) [reply R/meetup_query.R:3]: sorry, all your comments didnt become visible when I started! So I overran some of them!

File: R/meetup_query.R [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+#' (https://www.meetup.com/meetup_api/docs/).
+#' This function is convenient if you want to create
+#' a query to other Meetup API endpoints that we
+#' have not created functions for. If you do create
+#' a function that quieries a currently unsupported
```
Comment (maelle) [inline R/meetup_query.R]:
```suggestion
#' a function that queries a currently unsupported
```

**Comment by maelle** [comment]
Made one last comment, after this & document() feel free to merge, thanks a ton!

**Pull request merged**